app/controllers/scheduler_controller.rb

```diff
@@ -30,14 +31,28 @@ def destroy
   private
 
   def set_enrollment
-    @enrollment = Enrollment.where(user: current_user, course_instance_id: params[:course_instance_id]).first_or_create
+    course_instance_id =
+      if params[:course_instance_id].present?
+        params[:course_instance_id]
+      else
+        CourseInstance.search(params[:course_title], current_user, @search_date || Date.today)
+                      .first
+                      .try(:id)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.